envs/mujoco/test_env.py

```diff
@@ -0,0 +1,14 @@
+import sys
+
+import gym
+
+
+def main():
```

Comment:
Do we really need this file in the repo? I recommend either writing the test (like Python unit-test) or removing this altogether

Reply:
No, I will delete this.